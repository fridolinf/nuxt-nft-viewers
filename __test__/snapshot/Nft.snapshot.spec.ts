import NftPage from "../../pages/nft.vue";
import { RouterLinkStub, mount } from "@vue/test-utils";
import { useRoute } from "vue-router";
describe("Snapshot Nft Page", () => {
  test("Snapshot Nft page component", () => {
    const pages = mount(NftPage);
    expect(pages.html()).toMatchSnapshot();
  });
});
