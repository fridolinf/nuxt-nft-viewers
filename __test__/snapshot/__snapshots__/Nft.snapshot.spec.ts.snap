// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Nft Page Snapshot Nft page component 1`] = `
"<div class="flex flex-col justify-center items-center h-screen w-full">
  <div id="loading" class="flex-row gap-4 flex py-2"><span class="animate-ping bg-red-500 rounded-t-xl p-1"></span><span class="animate-ping bg-red-500 rounded-t-xl p-1"></span><span class="animate-ping bg-red-500 rounded-t-xl p-1"></span></div>
  <div class="flex flex-row gap-2">
    <p>Loading</p><span class="animate-bounce h-3 w-2">.</span><span class="animate-bounce h-3 w-2">.</span><span class="animate-bounce h-3 w-2">.</span>
  </div>
</div>"
`;
